pkg/interface/src/views/landscape/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { Box, Center } from '@tlon/indigo-react';
+
+import './css/custom.css';
+
+import { PatpNoSig, AppName } from '~/types/noun';
+import GlobalApi from '~/logic/api/global';
+import { StoreState } from '~/logic/store/type';
+import GlobalSubscription from '~/logic/subscription/global';
+import { Resource } from '~/views/components/Resource';
+import { PopoverRoutes } from './components/PopoverRoutes';
+import { UnjoinedResource } from '~/views/components/UnjoinedResource';
+import { GroupsPane } from './components/GroupsPane';
+import { Workspace } from '~/types';
+
+
+type LandscapeProps = StoreState & {
+  ship: PatpNoSig;
+  api: GlobalApi;
+  subscription: GlobalSubscription;
+}
+
+export default class Landscape extends Component<LandscapeProps, {}> {
+  componentDidMount() {
+    document.title = 'OS1 - Landscape';
+    // preload spinner asset
+    new Image().src = '/~landscape/img/Spinner.png';
+
+    this.props.subscription.startApp('groups')
+    this.props.subscription.startApp('chat')
+    this.props.subscription.startApp('publish');
+    this.props.subscription.startApp('graph');
+    
+  }
+
+  componentWillUnmount() {
+    this.props.subscription.stopApp('groups')
+    this.props.subscription.stopApp('chat')
+    this.props.subscription.stopApp('publish');
+    this.props.subscription.stopApp('graph');
+  }
```

Comment:
If you're in Landscape most of the time does it make sense to really open and close all of these just navigating to the home screen? Seems like we're finally at "subscribe to everything and never leave" life?

Reply:
u rite m8

Reply:
I can see a bunch of removals but don't we want to initiate these subscriptions in a central place? It looks like we'd just start already started subscriptions a lot.

Reply:
Pretty sure startApp is idempotent

Reply:
It just printfs iirc

Reply:
I mean console.logs, not like, in Dojo